tests/integration/ignite-new/new.test.js

```diff
@@ -0,0 +1,41 @@
+const test = require('ava')
+const execa = require('execa')
+const tempy = require('tempy')
+const jetpack = require('fs-jetpack')
+
+const IGNITE = `${process.cwd()}/bin/ignite`
+const APP_DIR = 'Foo'
+
+test.before(t => {
+  const tempDir = tempy.directory()
+  process.chdir(tempDir)
```

Comment:
Now creates tests in a temporary directory, like we should have been doing all along. :)